pichillilorenzo/flutter_inappwebview
# Disable / Enable Keyboard Input (Android & iOS)

## Connection with issue(s)

Resolve issue #1845 

Provides a way to enable / disable, as well as show and hide the keyboard on iOS and Android devices.

## Testing and Review Notes

See videos below. You may need to disable the `disableInputAccessoryView: true` in `InAppWebViewSettings` enabling / disabling does not remove this view by default.

1. Use the `InAppWebView` example on an Android or iOS device
2. Tap on an edit field in the webview, notice that the keyboard shows
3. Tap to hide the keyboard, notice that the keyboard will now be disabled whenever tapping on an edit text field
4. Tap to show the keyboard, notice that the keyboard is now enabled and reshown


## Screenshots or Videos

| Android | iOS |
| --- | --- |
<video src="https://github.com/user-attachments/assets/e3863e06-87a1-4e81-821c-83a6fbbc2ab2"/> | <video src="https://github.com/user-attachments/assets/552978b3-b2e2-4fab-9d80-d56215cc63d1" />
